File: CCL-example-java/src/main/java/hohserg/ccl/example/ClientPacketHandler.java
package hohserg.ccl.example;

import codechicken.lib.packet.ICustomPacketHandler.IClientPacketHandler;
import codechicken.lib.packet.PacketCustom;
import net.minecraft.client.Minecraft;
import net.minecraft.network.play.INetHandlerPlayClient;

public class ClientPacketHandler implements IClientPacketHandler {
    @Override
    public void handlePacket(PacketCustom packetCustom, Minecraft minecraft, INetHandlerPlayClient iNetHandlerPlayClient) {
        switch (packetCustom.getType()) {//Определяем тип пакета
            default:
        }
    }
}

File: CCL-example-scala/src/main/scala/hohserg/ccl/example/ClientPacketHandler.scala
package hohserg.ccl.example

import codechicken.lib.packet.ICustomPacketHandler.IClientPacketHandler
import codechicken.lib.packet.PacketCustom
import net.minecraft.client.Minecraft
import net.minecraft.network.play.INetHandlerPlayClient

class ClientPacketHandler extends IClientPacketHandler {
  override def handlePacket(packet: PacketCustom, mc: Minecraft, handler: INetHandlerPlayClient): Unit = {}
}
